### ISSUE TITLE
Implement facetting for `vis_value()`, `vis_binary()`, `vis_compare()`, `vis_expect()`, and `vis_guess()`.

### ISSUE THREAD

**Issue by njtierney** [ropensci/visdat#159]
See #78

**Comment by Generalized** [comment]
Definitely, this would be extremely useful. The problem with predefined faceting rather than relying on the external +facet_grid or facet_rap is already visible in the SurvMiner package: it works as long as the user fits the anticipated pattern: max 2 groups. If one has 3 grouping variables - the facet_wrap or facet_grid becomes an unreachable dream. Then "patchworking" or gridExtra may help, but it's definitely a non-trivial solution. Such 2+ grouping is a daily thing e.g. in medical research, clinical trials, where we have multiple visits and multiple treatment arms (2 grouping levels). If one wants to look at missing data patterns in each treatment arm at each visit - then it has to be done with manual recreation of these graphs with ggplot2.
